apps/explorer/src/pages/transaction-result/TransactionView.tsx

```diff
@@ -34,35 +26,7 @@ export function TransactionView({
     transaction: SuiTransactionBlockResponse;
 }) {
     const isMediumOrAbove = useBreakpoint('md');
-
-    // const txKindData = formatByTransactionKind(txKindName, txnDetails, sender);
-    // const txEventData = transaction.events?.map(eventToDisplay);
-
-    // MUSTFIX(chris): re-enable event display
```

Comment:
So showing events is entirely out of the transaction?

Reply:
it's not, but I figured we would either re-write or can always look back in git history for this code when we re-enable it